corehq/apps/reports/standard/cases/case_data.py

```diff
@@ -240,7 +239,6 @@ def _product_name(product_id):
 
             "default_properties_as_table": default_properties,
             "dynamic_properties": dynamic_data,
-            "dynamic_properties_as_table": self._get_dynamic_properties_as_table(dynamic_data, timezone),
```

Comment:
previously this used the same format for the DD version and the regular one - this switches that to two variables, which is a little easier to display as desired in the HTML